docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -157,16 +186,31 @@ To deserialize from UTF-8, call a <xref:System.Text.Json.JsonSerializer.Deserial
 
 The following behaviors apply when deserializing JSON:
 
+::: zone pivot="dotnet-5-0"
+
 * By default, property name matching is case-sensitive. You can [specify case-insensitivity](#case-insensitive-property-matching).
 * If the JSON contains a value for a read-only property, the value is ignored and no exception is thrown.
-* Constructors for deserialization:
-  - On .NET Core 3.0 and 3.1, a parameterless constructor, which can be public, internal, or private, is used for deserialization.
-  - In .NET 5.0 and later, non-public constructors are ignored by the serializer. However, parameterized constructors can be used if a parameterless constructor isn't available.
+* Non-public constructors are ignored by the serializer. However, parameterized constructors can be used if a parameterless constructor isn't available.
+* Deserialization to immutable objects or read-only properties is supported. See [Immutable types and Records](#immutable-types-and-records).
+* By default, enums are supported as numbers. You can [serialize enum names as strings](#enums-as-strings).
+* By default, fields are excluded. You can [include fields](#include-fields).
+* By default, comments or trailing commas in the JSON throw exceptions. You can [allow comments and trailing commas](#allow-comments-and-trailing-commas).
+* The [default maximum depth](xref:System.Text.Json.JsonReaderOptions.MaxDepth) is 64.
+
+    When you use System.Text.Json indirectly in an ASP.NET Core app, some default behaviors are different. For more information, see [Web defaults for JsonSerializerOptions](#web-defaults-for-jsonserializeroptions).
```

Comment:
This statement is also true for 3.1. Maybe add a corresponding note and inline details about case sensitivity and property naming policy?